WordPress/Classes/ViewRelated/Plans/PurchaseButton.swift

```diff
@@ -104,7 +103,7 @@ class PurchaseButton: RoundedButton {
                     self.cornerRadius = self.bounds.height / 2
                     self.titleLabel?.alpha = 0
                     
-                    self.layoutIfNeeded()
+                    self.superview?.layoutIfNeeded()
```

Comment:
I'm thinking that maybe `PurchaseButton` should be a view containing the button and activity indicator, instead of subclassing `UIButton`. Also, I think it's outside the scope of this PR, so I'd revert it for now and move the discussion to a new PR/issue.